<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbInputWindow</class>
 <widget class="QDialog" name="DbInputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>491</width>
     <height>481</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="short_name_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Long_Name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="short_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Module abbreviation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="long_name_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Module name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="semester_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semester:</string>
    </property>
   </widget>
   <widget class="QLabel" name="M_K_A_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Performance assessment:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Semester_select">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="M_K_A_select">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sofort Online Klausur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Termin Online Klausur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Assignment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laboratory report</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="start_date_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_date_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_min_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time in minutes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="note_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Note:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Note_Box">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1,0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1,3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1,7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2,0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2,3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2,7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3,0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3,3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3,7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4,0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>nicht bestanden</string>
     </property>
    </item>
   </widget>
   <widget class="QDateEdit" name="Start_dateEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="End_dateEdit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Time_Input">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ects_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ECTS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ECTS_Box">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="status_box">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>inactive</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>active</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>completed</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Module input window</string>
    </property>
   </widget>
   <widget class="QLabel" name="M_K_A_label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Performance assessment 2:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="M_K_A_select_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sofort Online Klausur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Termin Online Klausur</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Assignment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Laboratory report</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="submitbutton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
